content/asset-tracking/using-the-sdks.textile

```diff
@@ -120,12 +134,20 @@ The following example demonstrates establishing a connection using JWT authentic
 
 ```[android]
 val publisher = Publisher.publishers() // get the Publisher builder in default state
-    .connection(ConnectionConfiguration(Authentication.jwt(CLIENT_ID) { tokenParameters ->
+  .connection(ConnectionConfiguration(Authentication.jwt(CLIENT_ID) { tokenParameters ->
         // get JWT from your server
         getJWTFromAuthServer(tokenParameters); // customer implements this function
         }))
 ```
 
+```[swift]
+let options = ARTClientOptions()
+options.authUrl = NSURL(string: "http://my.website/jwt-auth")
+
+publisher = try? PublisherFactory.publishers() // get a Publisher builder
+  .connection(ConnectionConfiguration(options))
```

Comment:
This doesn't actually do anything yet (i.e. it **doesn't** establish a connection). `publishers()` actually returns a builder, so you need to finish it off with `.start()`, maybe the wording needs to change, or maybe change the code:

```
let options = ARTClientOptions()
options.authUrl = NSURL(string: "http://my.website/jwt-auth")

publisher = try? PublisherFactory.publishers() // get a Publisher builder
  .connection(ConnectionConfiguration(options))
  /* Put additional configuration here */
  .start()
```

Reply:
Changed